hack/generator/pkg/astmodel/identifier_factory.go

```diff
@@ -128,16 +146,23 @@ func simplifyName(context string, name string) string {
 }
 
 func sliceIntoWords(identifier string) []string {
+	// Trim any leading and trailing spaces to make our life easier later
+	identifier = strings.Trim(identifier, " ")
+
 	var result []string
 	chars := []rune(identifier)
 	lastStart := 0
 	for i := range chars {
 		preceedingLower := i > 0 && unicode.IsLower(chars[i-1])
 		succeedingLower := i+1 < len(chars) && unicode.IsLower(chars[i+1])
+		isSpace := unicode.IsSpace(chars[i])
 		foundUpper := unicode.IsUpper(chars[i])
 		if i > lastStart && foundUpper && (preceedingLower || succeedingLower) {
 			result = append(result, string(chars[lastStart:i]))
 			lastStart = i
+		} else if i > lastStart && isSpace {
+			result = append(result, string(chars[lastStart:i]))
+			lastStart = i + 1 // skip the space
```

Comment:
Do any of the test cases  cover this branch?

Reply:
Yes, I added a few that do